spec/models/teammate_spec.rb

```diff
@@ -1,4 +1,44 @@
 require 'rails_helper'
 
 describe Teammate do
+  describe ".ratings_count scopes count to current event" do
```

Comment:
I think this can simply be the name of the method:
`describe "#ratings_count" do`
...